client/src/transmitter/index.ts

```diff
@@ -66,21 +108,17 @@ async function initStreamData() {
   const { humanId, streamKey } = await res.json()
   console.log('streaming to', humanId)
 
-  _playbackId = humanId
-  _streamKey = streamKey
+  _playback_id = humanId
+  _stream_key = streamKey
 
-  const portStr = isLocalOrIp ? `:${port}` : ''
-  playbackUrl.innerText = `${protocol}//${hostname}${portStr}/${humanId}`
+  const portStr = is_local_or_ip ? `:${port}` : ''
+  _playback_url.innerText = `${protocol}//${hostname}${portStr}/${humanId}`
 }
 
-let record_frash_dim = false
-
-const minRetryThreshold = 60 * 1000 // 1 min
-
 type Transport = 'wrtc' | 'ws'
 const allTransports: Transport[] = ['wrtc', 'ws']
 
-function requested_transport() {
+function requestedTransport() {
```

Comment:
Why are the functions camels while the variables snakes? I could accept the uglier snake case if at least it was consistent, but this is just chaos now xD

Reply:
There's semantic meaning in the naming now: 

1) Variables are Snake Case;
2) Globals Variables start with "_";
3) Functions are Camel Case;

This separation is helpful for situations like [this](https://github.com/victorges/justcast.it/blob/ad392c0d1f7609aee998dff97c54e88476bb3f76/client/src/transmitter/index.ts#L140):

`const requested_transport = requestedTransport()`

If variables and functions followed the same standard, clever renaming would be necessary.

Reply:
(you asked for verbosity here it goes xD)

Yeah I got that pattern. I like having the `_` prefix on the global variables, it helps a lot and looks good to me even if the variables themselves were camel, like `_globalVariable` :feelsgood: 

But the different casing style for different kinds of identifiers just doesn't work out for me though, I'd have to change writing style/reading pattern-matcher depending on the type of identifier feels 'cumbersome'. Reminds me of hungarian notation, and we already have IDEs so we don't need that anymore xD

And not even the IDEs, but the use-cases of variables and functions leave it pretty clear whether the thing is a function or a variable when reading, and when it does not, it's because the function is being used as a variable instead. And in that case, what would we do? `const requested_transport_function = requestedTransport`? :nazare:

In that matter, the line of code you exemplified looks weird to me exactly because it contrasts the difference in naming style. I'd rather have the var named `transport`, `reqTransp` or the function be named `getRequestedTransport` (or even not have the variable, which in that specific code imo looks good too). I think the clever-naming work outweighs the benefit of having more uniform code.

Finally, regarding snakes itself, I think I read somewhere that they are generally easier for our brains to parse, right? I'd be OK with that if we decided now that it's going to be used everywhere. I just find it a bit troublesome to write, all the time having to reach that `_` key outside of the regular typing area in the keyboard while also holding shift key with the pinky finger doing like a Bolsonaro shotgun sign with my hand. I could live with that though, mostly by relying on the IDE autocomplete as well. Let's please keep it consistent between all identifier types tho 🙏 

My ideal (A):
1. Everything is `camelCase`;
2. Globals Variables start with `_`

But I think this is good too (B):
1. Everything is `snake_case`;
2. Globals Variables start with `_`

And I just thought of/remembered another one, having camel case + function differentiation + single casing (C):
1. Everything is `camelCase`;
2. Globals Variables start with `_`;
3. Functions start with uppercase, so `CamelCase`

WDYT

Reply:
Alright, multiple naming conventions on the same place seems to do more harm than good. Let's be standard and go with A.

I wouldn't be able to swallow C3.

Reply:
C3 is very C# xD